developer_menu/views/menu_view.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+<menuitem
+        id="conf_tech"
+        parent="base.menu_administration"
+        name="🧰"
```

Comment:
There is no images in menu item in odoo and OCA modules. Introducing it breaks that rule.

Reply:
It's not an image, it's a unicode char like emoji. Odoo use emojis in some context.
We are no more in ascii or latin world ;-)

The purpose of OCA is not to always follow the Odoo way in all contexts : there are many exmples

The advantage is not to be more accessible, it's to be more compact (in settings apps which is not practical), it's intentional.

This module is not supposed to be installed on production instance as its name tells then no confusion for users.

> I think it's better to move menu entries into the new développer menu, instead of duplicate.

I don't think on my side, it's a matter of taste but if someone miss this shortcut , it expects to find orginal menus.

Reply:
> The advantage is not to be more accessible

Well, i'm not blind and i'm not very aware of such topic. But IFAIK, set an emoji as the name of an item is not accessible for blind people. Also it can not be searched...
Généraly, using image introduces a lot of matter of taste. For exemple, the current wallet image doesn't mean "configuration" for me, but for you it does.

Reply:
Ok, what do you suggest instead of this wallet aka tool box ?

Reply:
"Développer"

Reply:
This i not a wallet but https://emojipedia.org/fr/bo%C3%AEte-%C3%A0-outils

![image](https://github.com/OCA/server-ux/assets/1853434/2230f926-ab3b-4a7d-a59f-8e463bf684e3)

Are you agree with  `🧰 Dev` ? Easy to find because of color and meaningful.